src/types.ts

```diff
@@ -99,6 +99,7 @@ export interface ArweaveWalletConnector {
 }
 
 export interface ArweaveDataProvider {
+  // add isAddress method
```

Comment:
viewblock has an API - it _may_ be worth using until we implement our own way to validate the type of id

```
GET https://api.viewblock.io/arweave/search?q=gh673M0Koh941OIITVXl9hKabRaYWABQUedZxW-swIA&network=mainnet
```

Reply:
Doesnt seem to be a publicly available api. Cant find any docs on it - but might be able to ask the viewblock team.
Considering we run the gateway, would this be a worthwhile type to add? Like a /id route, where it returns similarly to viewblocks api?

Reply:
maybe - not necessary for now. we'll open a separate ticket to explore our options here and if it's something ar.io gateways may be able to support in the future.